tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -31,16 +31,27 @@ func TestCreateDestroyWindowsInstance(t *testing.T) {
 	kubeconfig := os.Getenv("KUBECONFIG")
 	awscredentials := os.Getenv("AWS_SHARED_CREDENTIALS_FILE")
 	dir := os.Getenv("ARTIFACT_DIR")
+	privateKey := os.Getenv("SSH_PRIV_KEY_PATH")
 
 	// The e2e test uses Microsoft Windows Server 2019 Base with Containers image, m4.large type, and libra ssh key.
 	// The CI-operator uses AWS region `us-east-1` which has the corresponding image ID: ami-0b8d82dea356226d3 for
 	// Microsoft Windows Server 2019 Base with Containers.
 	cloud, err := cloudprovider.CloudProviderFactory(kubeconfig, awscredentials, "default", dir,
-		"ami-0b8d82dea356226d3", "m4.large", "libra")
+		"ami-0b8d82dea356226d3", "m4.large", "libra", privateKey)
 	assert.NoError(t, err, "error creating clients")
 
-	err = cloud.CreateWindowsVM()
+	credentials, err := cloud.CreateWindowsVM()
 	assert.NoError(t, err, "error creating Windows instance")
+	if credentials == nil {
+		t.Fatal("Credentials returned are empty")
```

Comment:
I have a question here, if this one is fatal, how could we know what is the error message?